spec/ns-onco-core/oncocore-genetics.txt

```diff
@@ -31,27 +31,32 @@ Conformance note: To be conformant to US Core, the code attribute must be a LOIN
 Implementation note: The performer of the test (organization or practitioner) should be included in the FHIR profile as the performer.actor.
 
 Conformance note: The category for this profile is set to GE (Genetics), a code from http://hl7.org/fhir/ValueSet/diagnostic-service-sections. This is contrary to the [Argonaut](http://www.fhir.org/guides/argonaut/r2/) and [US Core](http://hl7.org/fhir/us/core/STU3/index.html) specifications, which require the category 'LAB' in diagnostic reports containing laboratory results. This is assumed to be an oversight in the US Core and Argonaut specifications."
-// Property: SpecimenType 0..1
-// Property:          RegionStudied 0..*
+
                    ObservationSubjectOfRecord only Patient
-                   // US Core v3 requires an extensible binding to Code
-//                   Code from GeneticTestVS (extensible)
                    Code = LNC#81247-9 "Master HL7 genetic variant reporting panel"
                    Specimen substitute GeneticSpecimen
-//                   Category = V20074#GE
                    Observation
-//                   includes GeneticVariantFound 0..*
                    includes GeneticVariant 0..*
                    includes RegionStudied 0..*
                    // SpecimenType from GeneticSpecimenTypeVS (extensible)
 
 
-Entry:           RegionStudied
-Parent:            CodedNonLaboratoryObservation
-Description:       "A description of the coverage of the genome that was tested for variants."
-//Value:             string
+Entry:              RegionStudied
+Parent:             CodedNonLaboratoryObservation
+Description:        "Gene mutations that were tested for."
                     FocalSubject substitute RelatedCancerCondition
                     Code = LNC#53041-0 "DNA region of interest panel"
+                    BodyLocation 0..0
+                    Components.ObservationComponent
+                    includes GeneMutations 0..*
+
+Group:              GeneMutations
+Parent:             CodedObservationComponent
+Description:        "The symbolic representation of a genetic structural variant reported using HGVS nomenclature (gHGVS)."
+                   // or CodeableConcept from http://varnomen.hgvs.org  // this is required in the CGWG obs-variant FHIR profile.
+                    Code = LNC#36908-2 "Gene mutations tested for in Blood or Tissue by Molecular genetics method Nominal"
+                    DataValue from http://varnomen.hgvs.org (required)
+                    DataAbsentReason 0..0
```

Comment:
Suppose there is a standard test for a set of gene mutations, but one of them wasn't carried out, or the test went haywire for some reason. Maybe the DataAbsentReason should be left intact so it can be expressed that the gene was NOT successfully tested for.

Reply:
Not sure why I made DataAbsentReason 0..0.  I'll back out those changes so that all of the components include it.  However, the element in which the comment was made,`GeneMutations` with LNC#36903-2 should be removed entirely.  I kept it in there but it should not have been referenced.  This was instead further broken out into 2 separate variant components, one to describe the genomic variant and one to describe the amino acid (protein) variant.